_pytest/logging.py

```diff
@@ -337,8 +341,14 @@ def __init__(self, config):
         """
         self._config = config
 
+        auto_enable_log_stream = self._config.getoption('--log-cli-level')
+        if auto_enable_log_stream is not None:
+            # Explicitely enable log_cli is --log-cli-level was passed on the CLI
+            # It won't be automatically enabled is set on the ini file.
```

Comment:
@nicoddemus Are you aware of similar code (where command line options set sth automatically but not ini options) within other parts of pytest?

Reply:
The reason why it's only enabled automatically for the CLI option and not the INI option is because of the concerns raised on my initial PR.
Also, this PR automatically set's verbosity to 1 if `log_cli`is true...

Reply:
I'm more concerned that automatically enabling live-logs if log-cli-level is set conflicts with a valid use case (IMHO): user sets `log_cli_level=DEBUG` on `pytest.ini` so that they can just pass `-o log_cli=true` on the command-line and see DEBUG log messages live.

But to be honest I never used live logging, so I'm not sure even if this is an important/valid use case, so I will defer to @thisch the final verdict on this. 👍